source/howto/data/datum-alias.rst

```diff
@@ -0,0 +1,78 @@
+.. meta::
+    :description: This how to guide shows you how to create aliases for your data files and use them in your executions.
+
+.. _howto-datum-alias:
+
+Data aliases
+################################################
+
+You can create a data alias in your project and use the alias as the input for your executions.
+
+There are many different use cases for data aliases. For example:
+
+* Create an alias ``model-prod`` and point it to the version of a model that should be used for batch inference in production
+* Create an alias for ``train-images`` that points to the latest preprocessed dataset of a specific use case. The rest of the team can then point their inputs to this alias, instead of always searching for the latest version of the preprocessed data.
+
+Valohai keeps track of the change history of each alias, so you can understand when did the latest changes happen.
+
+Datum aliases are resolved during execution/task creation, which means copying executions will not change the data if the alias had been changed meanwhile.
+
+Create and update an alias
+-----------------------------------
+
+You can create or update an existing alias using the web UI, programatically when you create a new output, or using the Valohai APIs.
```

Comment:
I don't like these Grammarly commas... I would just do x or y instead of x, or y

Reply:
I've used oxford commas everywhere else so I think I'll just keep this for this page.